slippi-launcher/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenvNoCC
+, appimageTools
+, fetchurl
+, makeDesktopItem
+, copyDesktopItems
+, makeWrapper
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "slippi-launcher";
+  version = "2.10.5";
+
+  src = appimageTools.wrapType2 rec {
+    inherit pname version;
+
+    src = fetchurl {
+      url = "https://github.com/project-slippi/slippi-launcher/releases/download/v${version}/Slippi-Launcher-${version}-x86_64.AppImage";
+      hash = "sha256-zYxSVbxERLtz5/9IS8PUft6ZQw6kQtSUp0/KmmA/bmM=";
+    };
+
+    extraInstallCommands = ''
+      source "${makeWrapper}/nix-support/setup-hook"
+      wrapProgram $out/bin/slippi-launcher-${version} \
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform=wayland}}"
```

Comment:
does this flag have any effect when running in x11?

Reply:
Yes, it does. However, I was fortunately able to find a replacement that would not: `--ozone-platform-hint=auto`, which I have since replaced it with.